packages/web-app-external/src/App.vue

```diff
@@ -82,16 +74,20 @@ export default {
       })
     },
     appName() {
-      return this.$route.params.appName
+      return this.$route.query.app
     },
-    fileId() {
-      return this.$route.params.fileId
+    fileName() {
+      return this.currentFileContext.path.split('/').pop()
```

Comment:
You could `import { basename } from 'path'` and use that instead. Or we might even add a `fileName()` function to `FileContext` using that for convenience